integration_tests/models/_test_events.yml

```diff
@@ -203,3 +203,13 @@ models:
           expression: "{{ dbt_fullstory.json_value('user_properties', '$.good_key') }} = 'good_value'"
           config:
             where: event_id = 'user_total_sessions_test_3'
+      - dbt_utils.expression_is_true:
+          name: event_properties_is_correct
+          expression: "{{ dbt_fullstory.json_value('event_properties', '$.good_key') }} = 'good_value'"
```

Comment:
Looks like you will need `skip_parse=True` here as an argument to the `json_value` func. This was newly introduced in #17